vars/fetchCheriSDK.groovy

```diff
@@ -48,16 +48,14 @@ def call(Map args) {
         gitBranch = env.BRANCH_NAME
```

Comment:
The other half of this if is meant to be picking up the PR target branch. Is that not working for you?